tests/registry_disk_test.go

```diff
@@ -168,4 +171,34 @@ var _ = Describe("ContainerDisk", func() {
 			})
 		})
 	})
+
+	Describe("Starting with virtio-win", func() {
+		Context("with virtio-win as secondary disk", func() {
+			It("should boot and have the virtio as sata CDROM", func() {
+				vmi := tests.NewRandomVMIWithEphemeralDisk(tests.ContainerDiskFor(tests.ContainerDiskAlpine))
+				tests.AddEphemeralCdrom(vmi, "disk4", "sata", fmt.Sprintf("%s/virtio-container-disk:%s", tests.KubeVirtRepoPrefix, tests.KubeVirtVersionTag))
```

Comment:
This 'virtio-container-disk' image doesn't follow the convention we've been using for all our other container disks. We should be able to reuse the "ContainerDiskFor()" function here rather than building this image string directly.